libc/stdlib/abort.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+__attribute__((__noreturn__))
+void abort(void) {
+#if defined(__is_libk)
+	// TODO: Add proper kernel panic.
+	printf("kernel: panic: abort()\n");
+#else
+	// TODO: Abnormally terminate the process as if by SIGABRT.
```

Comment:
Not sure this will work. 🚀

Reply:
Panic? It's pretty easy to implement, I do that all the time